contracts/staking/PoSValidatorManager.sol

```diff
@@ -356,7 +371,9 @@ abstract contract PoSValidatorManager is IPoSValidatorManager, ValidatorManager
 
         Delegator memory delegator = $._delegatorStakes[delegationID];
         _unlock(delegator.weight, delegator.owner);
-        // TODO: issue rewards
+
+        _reward(delegator.owner, $._pendingDelegatorRewards[delegationID]);
+        delete $._pendingDelegatorRewards[delegationID];
```

Comment:
We should keep the ordering of these two operations consistent with the validator function.

Reply:
we could just put the delete in the `_reward` function too

Reply:
nvm this is for both PoS so should leave here